client.go

```diff
@@ -95,28 +88,43 @@ func Connect(endpoints string, tlsConfig *tls.Config) (*OvsdbClient, error) {
 	return nil, fmt.Errorf("failed to connect: %s", err.Error())
 }
 
-func newRPC2Client(conn net.Conn) (*OvsdbClient, error) {
+func newRPC2Client(conn net.Conn) (ovs *OvsdbClient, err error) {
```

Comment:
i'm against for named return values. Does it possible to not use it?

Reply:
@vtolstov but that is needed for the defer func(){} to work correctly in all cases.

```
func someFunc() (error) {
   var err error
   err := open()
   defer func() {
      if err != nil {
          // do soemthing
      }
  }

  if newErr :=DoSomething; newErr != nil {
     return fmt.Errorf("something failed %v", newErr)
  }
}
```

In the above code snippet, the defer() will see `err to be nil`.

If you had named error, then it will correctly in all cases.

Reply:
i'm prefer to add c.Close on two places inside this func and not use named ret values and defer

Reply:
@hzhou8 what you prefer?

Reply:
@vtolstov I think both approaches are ok for this simple function - I don't see any problem of using the defer function to handle the resource cleanup. Could you explain why you prefer adding c.Close() in two places instead of using defer?